interface/frontend/src/components/CandidateList.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+<div class="container">
+  <div class="row">
+    <div class="col-md-4">
+      <template v-if="candidates.length">
+        <div id="accordion">
+        <template v-for="(candidate, index) in candidates">
+          <candidate :candidate="candidate" :index="index"></candidate>
+        </template>
+      </div>
+      </template>
+      <template v-else>
+        <p class="card-text">No candidates available.</p>
+      </template>
+    </div>
+    <div class="col-md-8"></div>
+  </div>
+</div>
+</template>
+
+<script>
+import Candidate from './Candidate'
+
+export default {
+  components: { Candidate },
+  data () {
+    return {
+      candidates: []
+    }
+  },
+  created () {
+    this.fetchCandidates()
+  },
+  methods: {
+    fetchCandidates () {
+      this.$axios.get('/api/candidates').then(
```

Comment:
This should be a ``{% url "..." %}``?

Reply:
@lamby This won't work when developing with the dev server because the files are being served by the dev server instead of being rendered by Django. But it also wouldn't work with our current production set-up because the [index file](https://github.com/concept-to-clinic/concept-to-clinic/blob/master/interface/frontend/src/index.html) that gets served from Django only contains an empty `<app>` component into which the remainder of the frontend app gets rendered via JS. We can pass information from Django into the app by adding data as properties of the `<app>` component, but it seems a bit of a hack to me.

Reply:
Can you clarify this in the code itself? Otherwise this comment will be lost.. :)

Reply:
@lamby, added this comment to the [index file](https://github.com/concept-to-clinic/concept-to-clinic/pull/125/files#diff-f0219d0aabd1326ced417e7bd31ebc7d).